src/index.ts

```diff
@@ -1,12 +1,13 @@
 'use strict';
 import { ExpressAppConfig } from "./middleware/express.app.config";
 import { Oas3AppOptions } from "./middleware/oas3.options";
-import { OpenApiRequestHandler } from 'express-openapi-validator/dist/framework/types'
+import { OpenApiRequestHandler } from 'express-openapi-validator/dist/framework/types';
 
 export function expressAppConfig(
   definitionPath: string,
   appOptions: Oas3AppOptions,
-  customMiddlewares?: OpenApiRequestHandler[]
+  customMiddlewares?: OpenApiRequestHandler[], // This is the custom middleware that we want to pass to the ExpressAppConfig
```

Comment:
No need to change this now, but for future reference `middleware` is already plural, so you don't need to add an `s` :)